helix-term/src/config.rs

```diff
@@ -5,6 +5,7 @@ use crate::keymap::Keymaps;
 #[derive(Debug, Default, Clone, PartialEq, Deserialize)]
 pub struct Config {
     pub theme: Option<String>,
+    pub shell: Option<Vec<String>>,
```

Comment:
Shouldn't it be `String`?

Reply:
Allowing it to be input as a single string would require the string to be parsed as a shell command, which is what the shell is needed for in the first place.